app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate params[:session][:password]
```

Comment:
Dấu `&.` này có tác dụng gì vậy e?

Reply:
- Dấu &.(The Safe Navigation Operator): đây là cách viết an toàn thông thường để đảm bảo không có các exception không mong muốn bắn ra khi object bị nil.